File: Java/Week2/UniSystem/src/models/Teacher.java
package models;


public class Teacher
{
    public String teacherId, name, deptId;

    public Teacher(String input_line)
    {
        String[] p = input_line.split(",");
        teacherId = p[0].trim(); 
        name = p[1].trim(); 
        deptId = p[2].trim();
    }
}

File: PythonProject/Week 2/models/teacher.py
class Teacher:
    def __init__(self, teacher_id, name, email, dept_id, specialization, hire_date, salary):
        self.teacher_id = teacher_id
        self.name = name
        self.email = email
        self.dept_id = int(dept_id)
        self.specialization = specialization
        self.hire_date = hire_date
        self.salary = float(salary)

    def __repr__(self):
        return f"Teacher({self.teacher_id}, {self.name}, Dept:{self.dept_id})"
